<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JanelaAtualizar</class>
 <widget class="QDialog" name="JanelaAtualizar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Pesquisar</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Salvar</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="numeroLinha"/>
   </item>
   <item row="3" column="1">
    <widget class="QTextEdit" name="atualText"/>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Texto</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QComboBox" name="selecionarBox">
     <item>
      <property name="text">
       <string>Frequência</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Social</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Atividade</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Oral</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Português</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Arte</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Educação Física</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Matemática</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ciências</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>História</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Geografia</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Inglês</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Número</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Disciplina</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
